qml/MainScreen.qml

```diff
@@ -199,7 +204,7 @@ Item {
             addPanel(0, -1, topPanel)
 
             rightIndicator.visible  = Qt.binding(function() { return ((grid.toLeftAllowed   || (grid.currentVerticalPos == 1 && al.toLeftAllowed )) && !displayAmbient)})
-            leftIndicator.visible   = Qt.binding(function() { return ((grid.toRightAllowed  || (grid.currentVerticalPos == 1 && al.toRightAllowed)) && !displayAmbient)})
+            leftIndicator.visible   = Qt.binding(function() { return ((grid.toRightAllowed  || (grid.currentVerticalPos == 1 && al.toRightAllowed)) && (!displayAmbient || np.hasNew))})
```

Comment:
What's the point of keeping track of new/seen notifications? Wouldn't it be simpler to just have `!np.modelEmpty`?

Reply:
I tested it now like that, and it removes the indicator when notification is removed on phones side. You can then leave `NotificationsPanel.qml` unchanged.

Reply:
Thanks for your feedback! :smile: 
What you mean will indeed work. However with `!np.modelEmpty` you do not encompass when a user views a notification, as in the user swipes left, then does not dismiss it. This will make the indicator stay visible, while the user has seen the notification already.
What I want to achieve is a behavior that when a user has seen the notification but not dismissed it. There is no need to keep showing the indicator when the user has seen it but not dismissed it. This should improve the screen with respect to burn in problems.